extensions/PhabBugz/Extension.pm

```diff
@@ -31,14 +35,14 @@ sub template_before_process {
   return unless $file =~ /bug_modal\/(header|edit).html.tmpl$/;
 
   if (my $bug = exists $vars->{'bugs'} ? $vars->{'bugs'}[0] : $vars->{'bug'}) {
-    my $has_revisions = 0;
+    my $has_revisions         = 0;
```

Comment:
FWIW, This feels like an unrelated change.
Also, I wish we had an autoformatter to avoid unexpected spacing changes like this.